app/src/main/res/values/strings.xml

```diff
@@ -496,5 +496,5 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_fee_not_at_hours">Yes, but not…</string>
     <string name="quest_fee_hours_title">at the following times:</string>
     <string name="quest_housenumber_multiple_numbers">It has multiple house numbers</string>
-    <string name="quest_housenumber_multiple_numbers_description">"You can simply input comma-separated house numbers or as ranges,\nlike i.e. 1,3 or 2&#45;6."</string>
+    <string name="quest_housenumber_multiple_numbers_description">"You can simply enter comma-separated house numbers or ranges.\nFor instance 1,3 or 2-6."</string>
```

Comment:
maybe nitpick, but why choose comma as separator instead of semi-colon, which ismuch more commun I think in OSM?
https://wiki.openstreetmap.org/wiki/Semi-colon_value_separator

Reply:
"," is more common for housenumbers. Search for "," vs ";" on [taginfo](https://taginfo.openstreetmap.org/keys/addr%3Ahousenumber#values)